### ISSUE TITLE
Error: no plugin found for a rule: "remove-duplicate-interface-keys".

### ISSUE THREAD

**Issue by Sparklytical** [EvgenyOrekhov/eslint-config-hardcore#244]
This error is coming when I enable the hardcore config.

here is my eslint config -
```json
{
  "env": {
    "es2020": true,
    "es2021": true,
    "es6": true,
    "node": true
  },
  "extends": [
    "eslint:recommended",
    "galex",
    "hardcore",
    // "hardcore/node",
    "plugin:@typescript-eslint/recommended",
    "plugin:@typescript-eslint/recommended-requiring-type-checking",
    "plugin:eslint-comments/recommended",
    "plugin:import/typescript",
    "plugin:type-graphql/recommended",
    "prettier",
    "prettier/@typescript-eslint"
  ],
  "ignorePatterns": ["build", "node_modules"],
  "parser": "@typescript-eslint/parser",
  "parserOptions": {
    "ecmaVersion": 2021,
    "project": "./tsconfig.json",
    "sourceType": "module"
  },
  "plugins": ["eslint-comments", "import", "type-graphql"],
  "reportUnusedDisableDirectives": true,
  "root": true,
  "rules": {
    "@typescript-eslint/explicit-member-accessibility": "off",
    "class-methods-use-this": "off",
    "fp/no-class": "off",
    "fp/no-let": "off",
    "fp/no-mutating-methods": "off",
    "fp/no-mutation": "off",
    "func-style": [
      "error",
      "declaration",
      {
        "allowArrowFunctions": true
      }
    ],
    "import/extensions": "off",
    "import/no-cycle": "off",
    "import/no-unused-modules": "off",
    "import/order": [
      "error",
      {
        "alphabetize": {
          "order": "asc"
        },
        "newlines-between": "always"
      }
    ],
    "import/prefer-default-export": "off",
    "import/unambiguous": "off",
    "init-declarations": "off",
    "eslint-comments/disable-enable-pair": "off",
    "max-statements": [
      "error",
      15,
      {
        "ignoreTopLevelFunctions": true
      }
    ],
    "new-cap": "off",
    "no-param-reassign": "off",
    "no-shadow": "off",
    "no-warning-comments": "off",
    "node/file-extension-in-import": "off",
    "node/no-missing-import": [
      "error",
      {
        "allowModules": [],
        "resolvePaths": ["/path/to/a/modules/directory"],
        "tryExtensions": [".js", ".json", ".node", ".ts"]
      }
    ],
    "sort-imports": [
      "error",
      {
        "ignoreCase": true,
        "ignoreDeclarationSort": true
      }
    ],
    "type-graphql/invalid-nullable-output-type": "off",
    "unicorn/no-null": "off"
  }
}

```

**Comment by EvgenyOrekhov** [comment]
There was an issue on [Putout](https://github.com/coderaiser/putout)'s side, should be fixed in [eslint-config-hardcore@16.1.0](https://github.com/EvgenyOrekhov/eslint-config-hardcore/releases/tag/v16.1.0). Please try it and let me know how it goes.

**Comment by Sparklytical** [comment]
thanks for the quick response. i'll tell you in a minute
